bundles/net.tourbook/src/net/tourbook/database/TourDatabase.java

```diff
@@ -4866,6 +4867,93 @@ private void modifyColumn_Type(final String table,
       System.out.println();
    }
 
+   /**
+    * Do data updates concurrently with all available processor threads, this is reducing time
+    * significantly.
+    *
+    * @param concurrentUpdater
+    *           {@link IComputeNoDataserieValues} interface to concurrently update
+    *           tours
+    * @param tourId
+    *           Tour ID of the tour to be updated
+    * @return
+    */
+   public void performTourDataUpdate_ForAllTours(final long tourId,
```

Comment:
When a method is currently and is propably never used from other files, it should not be public

Reply:
Fixed